### ISSUE TITLE
Crash immediately after function is called to signIn or startAtLaunch

### ISSUE THREAD

**Issue by NathanFitchett** [freshplanet/ANE-Google-Play-Game-Services#39]
Hey everyone so I've been at this for days now. I am setting up the ANE using Flash CS6 and Flash Develop. I have the ANE connected to the library path in Flash CS6 also have it added to the library in Flash Develop. 

This is how I have everything setup in my startScreen Class:

![screenshot 65](https://cloud.githubusercontent.com/assets/6869776/7104504/175bcaf6-e09b-11e4-97c7-af1e1bd79ba4.png)

Now once I click the button on my Android Device the Application Crashes immediately after doing so here is how my XML is setup:

![screenshot 66](https://cloud.githubusercontent.com/assets/6869776/7104506/486d291e-e09b-11e4-953f-c39353c613ba.png)

I followed all the instructions as you can see. So i don't understand why it's crashing and not displaying anything. Please any help would be appreciated thanks!

**Comment by afief** [comment]
me too, i am trying use another AIR SDK version (15 & 17), but the problem still the same like this.

**Comment by cherniv** [comment]
Did someone find a solution??

**Comment by NathanFitchett** [comment]
Yes Finally got Google Play Services to work perfectly. No Problems so far. Anyone still need help?

**Comment by cherniv** [comment]
@NathanFitchett yes, my app still crushes when i'm tryin to SignIn or StartAtLaunch..

**Comment by UmberFunk** [comment]
@NathanFitchett  
It would be great if you share how you made it work.

**Comment by UmberFunk** [comment]
@cherniv 
I found a great guide. Here's sollution:
https://github.com/nitacawo/Google-play-freshplanet-Overlawled-examples 

There are 3 versions. I used Freshplanet example and replaced ANE with newest one from Freshplanet. 
Look into the -app.xml in exaple provided. You should add Vungle extension to your project. Thought author forgot to remove it and I didn't use it in my code util I get it that its needed to avoid crashes.

**Comment by goshdarn** [comment]
Hi nathan, could you tell us how you got it to work please?

**Comment by goshdarn** [comment]
@NathanFitchett 
could you tell us how you got it to work please? I am getting nowhere.

@UmberFunk
Freshplanet example does not have showLeaderboard method which I really need.

@cherniv
did you find a solution for the crashes?

**Comment by cherniv** [comment]
@goshdarn 
now it is working for me. i did two things: 1) stopped using an AdMob ANE 2) started the GP Services integration from scratch
